src/patternfly/components/Button/button.scss

```diff
@@ -2,18 +2,16 @@
 
 .pf-c-button {
   // Component
-  --pf-c-button--PaddingTop: var(--pf-global--spacer--xs);
-  --pf-c-button--PaddingRight: var(--pf-global--spacer--lg);
-  --pf-c-button--PaddingBottom: var(--pf-global--spacer--xs);
-  --pf-c-button--PaddingLeft: var(--pf-global--spacer--lg);
+  --pf-c-button--PaddingTop: #{pf-size-prem(6px)};
```

Comment:
I realize this isn't one of the spacers in the system, but I feel that the 6px needs to be a common variable since it's used in 3 different components and presumably they are the same for a reason.

Reply:
I agree. Do you have a suggestion as to what it should be called?

Reply:
did it! thanks for the suggestion :)